pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -1681,3 +1691,56 @@ func isDeviceOpened(deviceToDetach AttachedVolume, hostUtil hostutil.HostUtils)
 	}
 	return deviceOpened, nil
 }
+
+func cancelControlPlaneExpansion(pvc *v1.PersistentVolumeClaim, pv *v1.PersistentVolume, client clientset.Interface) (bool, error) {
```

Comment:
Add comment to this func that bool means we want to cancel.

Reply:
done.